labelbox/schema/slice.py

```diff
@@ -0,0 +1,51 @@
+from typing import List
+
+from labelbox.orm.db_object import DbObject
+from labelbox.orm.model import Field
+
+
+class Slice(DbObject):
```

Comment:
I think we decided on CatalogSlice, since ModelSlice will be introduced in the future?

Reply:
Ah just read your message https://labelbox.slack.com/archives/C03FSV0K6MP/p1665642677853339?thread_ts=1665598292.363279&cid=C03FSV0K6MP.

If this `Slice` class is a parent class, I'm okay with it